### DISCUSSION TITLE
Hide subtotal in minicart based on customer group

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#164] [1. Help]
Hello,

I have a requirement of hiding subtotal from minicart. I need to add a condition to see if the customer group is "retailer" and hide the subtotal.
Can this be achieved in vendor/swissup/module-breeze/view/frontend/web/js/components/minicart.js?

I can't really figure out how to get customer details in minicart.js.

**Comment by vovayatsyuk** [comment]
Have you considered using Magento's built-in "Minimum Advertised Price" feature? It allows to do the same without coding.

It will hide the price if it's less than Minimum Advertised Price. Since Magento allows you to set prices per customer group, you can set prices for "retailer" less than "Minimum Advertised Price".

And here is what it looks like:

Description | Screenshot
------------|--------------
Config | ![image](https://github.com/breezefront/community/assets/306080/736d3cb3-1954-4960-8770-6797fcac28f8) 
Product info | ![image](https://github.com/breezefront/community/assets/306080/1bc4050f-41a1-47b1-92ec-a0fc33bda168) 
Frontend | ![image](https://github.com/breezefront/community/assets/306080/ed420fb4-808e-4aa5-8e53-c0cf0c0295a9)
